src/client/scripts/room.js

```diff
@@ -52,13 +52,14 @@ Template.room.events({
         var tags = event.target.form[0].value.split(",");
 
         for(var i = 0; i < tags.length; i++){
-            tags[i] = tags[i].toLowerCase();
+            tags[i] = tags[i].trim().toLowerCase();
```

Comment:
the javascript way to do this is:
tags = tags.map(function(element) {
  return element.toLowerCase().trim();
});
see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map for more info, its pretty cool.  Don't think it matters what is used, I believe map is faster over large domains, but for what we are doing it shouldn't matter.